lib/l10n/app_en.arb

```
@@ -850,9 +850,11 @@
   "aiResponseTypeTaskSummary": "Task Summary",
   "aiSettingsAddedLabel": "Added",
   "aiSettingsAddModelButton": "Add Model",
+  "aiSettingsAddModelButton": "Add model",
```

RuboCop (1.86.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)